### PR TITLE
Added simple view for single fields

### PR DESCRIPTION
**Pull request by vcg-development**
This PR adds a minimalistic view if you just need a single repeatable field instead of a group of fields.

simpleview

### CODE REVIEW

**Review by mireo91 — CHANGES_REQUESTED** [review]
From this commit I see only changes on Readme
nothing else was changed

**Comment by vcg-development** [comment]
@mireo91 I'm sorry. I forgot to push the changes. It should be visible now. :)

File: Resources/Private/Backend/RepeatableField/src/View/Repeatable.js [hunk 1]
```diff
@@ -292,34 +292,59 @@ export default class Repeatable extends PureComponent {
   createElement = (idx) => {
     const { options } = this.props;
     const { allowRemove, currentValue } = this.state;
-		const isPredefined = options.predefinedProperties && options.predefinedProperties[idx]?true:false;
+		const isPredefined = options.predefinedProperties && options.predefinedProperties[idx] ? true : false;
```
Comment (jonnitto) [inline Resources/Private/Backend/RepeatableField/src/View/Repeatable.js:295]: Please use spaces
```suggestion
     const isPredefined = options.predefinedProperties && options.predefinedProperties[idx] ? true : false;
```

**Review by jonnitto — CHANGES_REQUESTED** [review]

**Comment by vcg-development** [comment]
@jonnitto Done

File: Resources/Private/Backend/RepeatableField/src/View/Repeatable.js [hunk 2]
```diff
@@ -292,34 +292,59 @@ export default class Repeatable extends PureComponent {
...
+                icon="trash"
+              />
+            ) : (
+              ""
+            )}
+          </div>
+          {this.getProperties(idx)}
+        </div>
+      );
+    }
```
Comment (jonnitto) [inline Resources/Private/Backend/RepeatableField/src/View/Repeatable.js:347]: You can also remove this `else` block, as the `if` before has a `return` statement

```suggestion
    }
 
    return (
        <div className={style.wrapper}>
          <div class={style.buttons}>
            {!isPredefined && options.controls.move && currentValue.length > 1 ? (
              <DragHandle />
            ) : (
              ""
            )}
            {!isPredefined && options.controls.remove && allowRemove ? (
              <IconButton
                onClick={() => this.handleRemove(idx)}
                className={style.delete}
                icon="trash"
              />
            ) : (
              ""
            )}
          </div>
          {this.getProperties(idx)}
        </div>
      );
```

**Comment by vcg-development** [comment]
@jonnitto Is there still something I can improve or is it ready to be merged?

**Comment by jonnitto** [comment]
No, we've to wait for @mireo91

**Comment by jonnitto** [comment]
@vcg-development The fields looks a bit unaligned. I think there is some more CSS work needed…

https://github.com/mireo91/RepeatableFields/assets/4510166/dd57342c-a688-4221-887a-baf35b8967d6

**Comment by vcg-development** [comment]
> @vcg-development The fields looks a bit unaligned. I think there is some more CSS work needed…
> 
>  CleanShot.2024-06-18.at.12.55.08.mp4

We'll have a look at it.

**Comment by vcg-development** [comment]
@jonnitto I just tried to fix the problem but it's looking good on my end. Can you please send me an example yaml property so I can test it?

**Comment by jonnitto** [comment]
@vcg-development If it ok for you I would integrate this into this PR #28

**Comment by vcg-development** [comment]
Sure. Thank you!

**Comment by jonnitto** [comment]
> Sure. Thank you!

Already done 😉

**Comment by jonnitto** [comment]
@mireo91 When #28 is merged, you can close this one

**Comment by mireo91** [comment]
@vcg-development new release v2.2.2 should contain your modifications https://github.com/mireo91/RepeatableFields/releases/tag/v2.2.2
Many thanks and sorry for delays

**Pull request closed without merging**